product_stock_state/models/res_company.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2017-Today GRAP (http://www.grap.coop).
+# @author Sylvain LE GAL <https://twitter.com/legalsylvain>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    stock_state_threshold = fields.Float(default=10, digits="Stock Threshold")
```

Comment:
Shouldn't this use decimal precision ?

Reply:
It is, isn't it ?  `digits="Stock Threshold"` :smile_cat: